ground/gcs/src/libs/tlmapcontrol/mapwidget/tlmapwidget.h

```diff
@@ -388,6 +389,7 @@ namespace mapcontrol
         void setSelectedWP(QList<WayPointItem *> list);
 
         void setWindVelocity(double windVelocity_NED[3]);
+        internals::PointLatLng dummy() {return internals::PointLatLng();}
```

Comment:
Nope, it was to make sure the dynamic library exported the functions it imported from the static lib. It is a common workaround. But since I dropped the static/dynamic arrangement and made it into a single dynamic lib this is not needed anymore.